json/data.json

```diff
@@ -0,0 +1,334 @@
+{
+  "products": [
+    {
+      "id":1,
+      "discount": "15",
+      "url": "img/potato.png",
+      "name": "New Potato (Aloo)",
+      "sourcedAt": "1 am",
+      "quantity": "1kg",
```

Comment:
don't store units along with quantity, how will you perform calculation, just keep the value